participant/tests.py

```diff
@@ -1,3 +1,121 @@
-from django.test import TestCase
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
```

Comment:
Lines 2, 5 and 6 should be deleted as they are unused imports.

Reply:
Thanks a lot! I hadn't realized that I had imports that I wasn't using, plus the 'Sonarlint' extension I use for vscode didn't tell me that they weren't used.